src/clj/hakukohderyhmapalvelu/handler.clj

```diff
@@ -88,6 +90,21 @@
    session-client/wrap-session-client-headers
    (session-timeout/create-wrap-absolute-session-timeout config)])
 
+(def datetime-format "yyyy-MM-dd'T'HH:mm:ss")
+(def datetime-parser (f/formatter datetime-format (t/default-time-zone)))
+
+(defn- parseDatetime
+  ([datetimeStr fieldDesc]
+   (parseDatetime datetimeStr fieldDesc nil))
+  ([datetimeStr fieldDesc default]
+   (if-not (nil? datetimeStr)
```

Comment:
Tässä ei tarvinne ottaa huomioon falsea, joten ei tarvitse tarkistaa näin hankalasti
```suggestion
   (if datetimeStr
```